src/index.html

```diff
@@ -11,7 +11,139 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <!-- <h1>HTML Form</h1> -->
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post" 
+      class="form">
+      <fieldset>
+        <legend>Personal information</legend>
+        <label class="label">
+          <div class="surname"> 
+            Surname:
+            <input type="text" name="surname" 
+              autocomplete="off"
+            >
+          </div>
+        </label>
+        <label class="label">
```

Comment:
according to code styling add an empty line between _multiline_ siblings
```suggestion
        </label>
        
        <label class="label">
```